unittests/allplatformstests.py

```diff
@@ -5010,3 +5010,24 @@ def test_c_cpp_stds(self):
             # The first supported std should be selected
             self.setconf('-Dcpp_std=c++11,gnu++11,vc++11')
             self.assertEqual(self.getconf('cpp_std'), 'c++11')
+
+    def test_rsp_support(self):
+        env = get_fake_env()
+        cc = detect_c_compiler(env, MachineChoice.HOST)
+        if cc.linker.id in {
+            'ld.bfd',
+            'ld.gold',
+            'ld.lld',
+            'ld.mold',
+            'ld.qcld',
+            'ld.wasm',
+            'link',
+            'lld-link',
+            'mwldarm',
+            'mwldeppc',
+            'optlink',
+            'xilink',
+        }:
+            assert cc.linker.get_accepts_rsp() == True
+        else:
+            assert cc.linker.get_accepts_rsp() == False
```

Comment:
```suggestion
        has_rsp = cc.linker.id in {
            'ld.bfd',
            'ld.gold',
            'ld.lld',
            'ld.mold',
            'ld.qcld',
            'ld.wasm',
            'link',
            'lld-link',
            'mwldarm',
            'mwldeppc',
            'optlink',
            'xilink',
        }
        self.assertEqual(cc.linker.get_accepts_rsp(), has_rsp)
```

Reply:
Why? `pytest` assert syntax is cleaner, and gives better error messages.

Reply:
We don't use pytest, we use unittest, and allow running with pytest when possible.

Reply:
Is this about asserting the difference between "is True" vs "is the current value of has_rsp"?